cpg_infra/driver.py

```diff
@@ -331,11 +337,18 @@ def setup_hail_batch_billing_project_members(self):
                 _group_members = self.group_provider.resolve_group_members(
                     dataset_cloud_infra.analysis_group
                 )
-                hail_batch_username = [
+                hail_batch_usernames = {
```

Comment:
This is casted to a set later in func "_add_member_to_billing_project", wonder if it would be better to construct it as set at first place instead of dict and then set(), set.update() on line 346 should work as is.